client/core/core.go

```diff
@@ -1796,6 +1785,22 @@ func (c *Core) Exchange(host string) (*Exchange, error) {
 	return dc.exchangeInfo(), nil
 }
 
+// ExchangeMarket returns the market with the given base and quote assets at the
+// given host. It returns an error if no market exists at that host.
+func (c *Core) ExchangeMarket(host string, base, quote uint32) (*Market, error) {
```

Comment:
Not entirely necessary, but a nice convenience compared to using the `Exchange` method and iterating through its markets.